modules/programs/command-not-found/command-not-found.pl

```diff
@@ -1,4 +1,4 @@
-#! @perl@/bin/perl -w
+#!/usr/bin/env -S @perl@/bin/perl -w
 
 use strict;
```

Comment:
Another alternative, that I think may be better:

```suggestion
#!@perl@/bin/perl

use strict;
use warnings;
```